### ISSUE TITLE
Untracked files in stashes are not shown

### ISSUE THREAD

**Issue by astromind** [JetpackDuba/Gitnuro#88]
**Describe the bug**
When viewing a stash, Gitnuro does not show untracked files in `Files changed` panel.
_(Not sure whether it should be a bug or feature request, but I definitely wouldn't expect this behavior)_

**To Reproduce**
1. Create a stash with untracked files. For example, execute following commands:
```
mkdir stash_test
cd stash_test/
git init
touch hello.txt
git add hello.txt 
git commit --message="initial commit"  
echo "hello world" >> hello.txt
echo "hello world" >> newfile.txt  
git stash save -u
```

2. Open the `stash_test` folder in Gitnuro. Select the stash on the left.
On the right panel you'll see only 1 file (`hello.txt`).

**Expected behavior**
Both files should be shown on the right panel:
![Stash_expected](https://user-images.githubusercontent.com/4924097/219975812-9e8fdfa0-89b7-4cdf-8063-d6b2ca7968b6.png)

As `git stash show -u` output shows:
```
 hello.txt   | 1 +
 newfile.txt | 1 +
 2 files changed, 2 insertions(+)
```

PS. Setting ` stash.showIncludeUntracked ` option does not affect the behavior.

**Screenshots**
Observed result:
![Stash](https://user-images.githubusercontent.com/4924097/219975614-724db624-802d-4716-b47e-5e6310337837.png)

**Desktop (please complete the following information):**
 - OS: Linux
 - Version 1.1.1

**Comment by JetpackDuba** [comment]
Ah yes, I've found this issue in the past but forgot to look into it with more detail. The problem is when stashing untracked files that are not index. When stashing from Gitnuro I add the new files to the index before stashing.

I'll look into it, thanks!

**Comment by JetpackDuba** [comment]
I've got a fix for it but I would really like to test it more, as I had some weird crashes under very specific circumstances, so I'll have the fix available for relase 1.3.0 rather than 1.2.1.

Thanks again for your report!
